webapi/src/test/scala/org/knora/webapi/e2e/CORSSupportE2ESpec.scala

```diff
@@ -51,30 +51,31 @@ class CORSSupportE2ESpec extends E2ESpec(CORSSupportE2ESpec.config) {
   "A Route with enabled CORS support" should {
 
     "accept valid pre-flight requests" in {
-      val request = Options(baseApiUrl + s"/admin/projects") ~> Origin(exampleOrigin) ~> `Access-Control-Request-Method`(
-        GET)
+      val request =
+        Options(baseApiUrl + s"/admin/projects") ~> Origin(exampleOrigin) ~> `Access-Control-Request-Method`(GET)
       val response: HttpResponse = singleAwaitingRequest(request)
       response.status shouldBe StatusCodes.OK
       response.headers should contain allElementsOf Seq(
         `Access-Control-Allow-Origin`(exampleOrigin),
-        `Access-Control-Allow-Methods`(List(GET, PUT, POST, DELETE, HEAD, OPTIONS)),
+        `Access-Control-Allow-Methods`(List(GET, PUT, POST, DELETE, PATCH, HEAD, OPTIONS)),
         `Access-Control-Max-Age`(1800),
         `Access-Control-Allow-Credentials`(true)
       )
     }
 
-    "reject requests with invalid method" in {
-      val request = Options(baseApiUrl + s"/admin/projects") ~> Origin(exampleOrigin) ~> `Access-Control-Request-Method`(
-        PATCH)
-      val response: HttpResponse = singleAwaitingRequest(request)
-      responseToString(response) shouldEqual "CORS: invalid method 'PATCH'"
-      response.status shouldBe StatusCodes.BadRequest
-    }
+//    "reject requests with invalid method" in {
+//      val request = Options(baseApiUrl + s"/admin/projects") ~> Origin(exampleOrigin) ~> `Access-Control-Request-Method`(
+//        PATCH)
+//      val response: HttpResponse = singleAwaitingRequest(request)
+//      responseToString(response) shouldEqual "CORS: invalid method 'PATCH'"
+//      response.status shouldBe StatusCodes.BadRequest
+//    }
```

Comment:
can this be removed?

Reply:
You are right. It's probably confusing in the future to have uncommented code blocks. I removed it.